episodes/03-create.md

```diff
@@ -726,11 +726,15 @@ $ cp minotaur.dat unicorn.dat basilisk.dat
 
 ## Solution
 
-When `cp` is given two arguments and the second is a destination directory
-`cp` copies the files to the destination directory.
+When `cp` is given three or more arguments,
+the last argument is interpretted as the "destination
+directory".
+In that case, `cp` copies all files (or directories)
+into that "destination directory".
+If the final argument is not a directory,
+it's unclear what should be copied where so 
+`cp` throws an error such as the one below.
```

Comment:
```suggestion
If the final argument is not a directory,
the behavior is undefined, so 
`cp` throws an error such as the one below.
```

Reply:
I think "undefined behavior" might be a phrase that learners are unfamiliar with. I'll try to think of a way to say the same thing in simpler words.